tusk_11py.3.py

```diff
@@ -0,0 +1,14 @@
+class CustomContextManager:
+    def __enter__(self):
+        print("=" * 10)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("=" * 10)
```

Comment:
You need to print exception text here, if there is some exception. To check it, you can created a conditional expression, where check if `exc_val` is not empty and if it is not - print its value:
```
if exc_val:
    print(exc_val)
```

to avoid raising of that exception, add `return True` in the end instead of `return self`

Reply:
so, in the end, your method should look like this:
```
if exc_val:
    print(exc_val)
print('=' * 10)
return True
```